src/constants.js

```diff
@@ -10,11 +10,18 @@ export default {
   PRODUCT_FILTER_LIST: [
     { key: 'hide_price_count_gte_1', value: 'Hide products with prices' },
   ],
+  PRICE_FILTER_LIST: [
+    { key: 'show_last_month', value: 'Show Prices from Last 30 Days' },
```

Comment:
```suggestion
    { key: 'only_last_30d', value: 'Only prices for the last 30 days' },
```

Reply:
btw these order / filter labels are not translated (seperate PR)

Reply:
Yes, I wanted to discuss on this first since it would mean adding translation in constant.js

Reply:
Or doing something similar as `routes.js` ?

Reply:
Will create another PR